src/aiortc/contrib/media.py

```diff
@@ -185,12 +181,11 @@ async def stop(self):
         self.__tracks = {}
 
 
-def place_frame_in_video_queue(frame, loop, video_track, container):
+def place_frame_in_video_queue(frame, frame_index, loop, video_track, container):
     """ place the attached frame in the video track queue after stamping it 
     """
     frame_time = frame.time
-    frame_index = frame.index
-    frame = stamp_frame(frame, frame.index, frame.pts, frame.time_base)
+    frame = stamp_frame(frame, frame_index, frame.pts, frame.time_base)
```

Comment:
why is this data not available now? is this because of interpolation?

Reply:
yes, converting NumPy array to av.VideoFrame results in a frame with index=0.